packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1087,9 +1087,7 @@ class DefaultTransitionDelegate<T> extends TransitionDelegate<T> {
 /// The default value of [Navigator.routeTraversalEdgeBehavior].
 ///
 /// {@macro flutter.widgets.navigator.routeTraversalEdgeBehavior}
-const TraversalEdgeBehavior kDefaultRouteTraversalEdgeBehavior = kIsWeb
-  ? TraversalEdgeBehavior.leaveFlutterView
-  : TraversalEdgeBehavior.closedLoop;
+const TraversalEdgeBehavior kDefaultRouteTraversalEdgeBehavior = TraversalEdgeBehavior.parentScope;
```

Comment:
Is this the right default for web?

Given the code you added in `app.dart`, I would have expected this to be:

```dart
const TraversalEdgeBehavior kDefaultRouteTraversalEdgeBehavior = kIsWeb
  ? TraversalEdgeBehavior.leaveFlutterView
  : TraversalEdgeBehavior.parentScope;
```

Reply:
This is to make sure the nested navigator would work in web.

cc @yjbanov correct me if i am wrong.

I think this should only be set to `leaveFlutterView` for the top-most focus scope, so that it can traverse focus to browser buttons